doc/src/sgml/gin.sgml

```diff
@@ -870,7 +880,7 @@ GINは<quote>部分一致</quote>問い合わせをサポートすることが
      less than when it is not.  But for very large updates it may still be
      best to drop and recreate the index.
 -->
-<productname>PostgreSQL</productname> 8.4から遅延インデックス作成が使用されるため、この勧告は必要性が薄れました。
+GINでは<literal>fastupdate</literal>が有効であるため、この勧告は必要性が薄れました。
```

Comment:
以前は接続詞がsinceでしたが、whenに変わってますので
```
GINでは<literal>fastupdate</literal>が有効であるため、
```
は
```
<acronym>GIN</acronym>への<literal>fastupdate</literal>が有効である場合、
```
などの訳が良いように思います。
※GINについているようなタグも省略は無しでお願いします。

Reply:
@KenichiroTanaka 
上記、指摘いただいた箇所を修正いたしました。
文末も日本語が自然になるようにしています。